src/nimib/jsutils.nim

```diff
@@ -136,9 +136,18 @@ proc genCapturedAssignment(capturedVariables, capturedTypes: seq[NimNode]): tupl
       import std/json
     for (cap, capType) in zip(capturedVariables, capturedTypes):
       let placeholder = gensym(ident="placeholder")
+      var newSym: NimNode
+      if "`gensym" notin cap.strVal:
+        newSym = gensym(NimSymKind.nskLet, ident=cap.strVal)
+        # add to tab[cap] = cap.gensym?
+        tabMapIdents[cap.strVal.nimIdentNormalize] = newSym.repr.ident
+      else:
+        newSym = cap
+      echo "In: ", cap.repr, " Out: ", newSym.repr
```

Comment:
Found a wild debug line :open_mouth:

Reply:
Hehe there are even more one like that, that got pushed in the release I realized :rofl: I'll clean them all up